scripts/resolve-workspace-deps.mjs

```
@@ -30,3 +34,24 @@ for (const dir of pkgDirs) {
+  for (const depType of ['dependencies', 'devDependencies', 'peerDependencies', 'optionalDependencies']) {
+    if (!pkg[depType]) continue;
+    for (const [dep, ver] of Object.entries(pkg[depType])) {
+      if (typeof ver === 'string' && ver.startsWith('workspace:')) {
+        unresolved.push(`${pkgName} -> ${depType}.${dep} (${ver})`);
+      }
+    }
+  }
+}
+
+if (unresolved.length > 0) {
+  console.error('Unresolved workspace dependencies remain:');
```

ESLint

[error] 'console' is not defined.

(no-undef)